app/Http/Controllers/SearchController.php

```diff
@@ -17,13 +18,65 @@ public function __construct()
     }
 
     public function searchMovie(Request $request)
-    {
+    {   
+        
+        if ($request->searchoption === "genre") {
+            $genreId = '';
+            switch ($request->search) {
+                case "Action":
```

Comment:
It seems the genre search is case sensitive, so when users search for a genre it has to be for example exactly "Action"  and not "action" or "ACTION". We can try making this case insensitive.

Reply:
Will look up workarounds, obviously the function should be flexible enough to allow a user to write capitalized as well as not